weave/Scripts/Multiplayer/Manager.cs

```diff
@@ -0,0 +1,204 @@
+using Godot;
+using System;
+using SIPSorcery.Net;
+using Newtonsoft.Json;
+using Weave.InputSources;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Net.WebSockets;
+using System.Text;
+
+namespace Weave.Multiplayer;
+
+public partial class Manager : Node
```

Comment:
We should probably have a more descriptive name? Perhaps NetworkClientManager? or something like that